frontend/components/Accounts/Modals/Verification.tsx

```diff
@@ -36,7 +36,7 @@ const VerificationForm = (props: VerificationFormProps) => {
                 closeFunc();
                 mutate();
                 addToast("Verification success!");
-            } catch (e) {
+            } catch (e: any) {
```

Comment:
Just a thought - there's a better annotation for sure, but it is sort of up to us to define it properly (you can do so by actually making verifyContact fail and figuring out the shape of what it throws. Is there a better way to enforce that it throws a specific type? (FWIW I'm not sure if we can do better than annotating the right type ourselves, but just a thought experiment.